app/code/core/Mage/Adminhtml/Block/Widget/Form.php

```diff
@@ -197,7 +197,7 @@ protected function _setFieldset($attributes, $fieldset, $exclude=array())
                         'label'     => $attribute->getFrontend()->getLabel(),
                         'class'     => $attribute->getFrontend()->getClass(),
                         'required'  => $attribute->getIsRequired(),
-                        'note'      => $this->escapeHtml($attribute->getNote()),
+                        'note'      => $this->escapeHtml($attribute->getNote())
```

Comment:
```suggestion
                        'note'      => $this->escapeHtml($this->__($attribute->getNote()))
```

Reply:
@kiatng The attribute at this stage is already translated. I followed the same business logic of the "FrontendLabel" attribute you find a few lines above (app/code/core/Mage/Adminhtml/Block/Widget/Form.php:197).

The attribute translates to: app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tab/Account.php:71 
```
...
$attributes = $customerForm->getAttributes();
 foreach ($attributes as $attribute) {
    /* @var $attribute Mage_Eav_Model_Entity_Attribute */
    $attribute->setFrontendLabel(Mage::helper('customer')->__($attribute->getFrontend()->getLabel()));
    $attribute->setNote(Mage::helper('customer')->__($attribute->getNote()));
    $attribute->unsIsVisible();
}
...
```

or 

app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tab/Addresses.php: 133
```
...
foreach ($attributes as $attribute) {
    /* @var $attribute Mage_Eav_Model_Entity_Attribute */
    $attribute->setFrontendLabel(Mage::helper('customer')->__($attribute->getFrontend()->getLabel()));
    $attribute->setNote(Mage::helper('customer')->__($attribute->getNote()));
    $attribute->unsIsVisible();
}
...
```

Reply:
In that case, I suggest not to change Form.php at all:

```suggestion
	                  'note'      => $this->escapeHtml($attribute->getNote()),
```